lib/routes.js

```diff
@@ -1,3 +1,9 @@
+
```

Comment:
Yes, lets do `strip` in case it will not delete line break at the last line or `lstrip` otherwise.